tests/unit/algos/list/delimiterMaster.spec.js

```diff
@@ -59,6 +153,63 @@ function getListingKey(key, vFormat) {
             assert.strictEqual(delimiter.skipping(), SKIP_NONE);
         });
 
+        it('should list all subprefixes when handleStreak is applied as in RepdServer', () => {
+            const prefix = '_Common-Prefix/';
+            const state = createStreakState();
+            const maxKeys = 1000000;
+            const delimiter = new DelimiterMaster({
+                maxKeys,
+                prefix,
+                delimiter: '/',
+                startAfter: '',
+                continuationToken: '',
+                v2: true,
+                fetchOwner: false }, fakeLogger, vFormat);
+            
+            [...generateKeys(maxKeys)].forEach(ob => {
+                const {key, isDeleteMarker, canSkip} = ob;
+                // simulate not doing a raw listing when key < params.gte
+                // this represents a key not reaching the read stream from leveldb
+                if (state.params.gte && key < state.params.gte) {
+                    return;
+                }
+
+                if (!key.includes(VID_SEP)) { // master key
+                    const version = new Version({ isDeleteMarker });
```

Comment:
in v1 there is no delete marker key in the master prefix, so we would have to not include them in the listing.

By the way, we should either fix v1 as well or open a ticket for it, if it is not fixed in v1. We can have the test ready for v1 but skip it in the meantime.

Reply:
If there is no such issue with v1, it's fine then with just running the same test on v1.

Reply:
I moved the test to only run under v0; v1 doesn't have this bug and the code change only affects v0 (v1 has separate logic for filtering)